### PR TITLE
docs(apig): update fields description

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [X] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/resources/apig_api.md [hunk 1]
```diff
@@ -63,9 +63,10 @@ The following arguments are supported:
   The valid values are **Public** and **Private**.
 
 * `name` - (Required, String) Specifies the API name.  
-  The valid length is limited from can contain `3` to `64`, only Chinese and English letters, digits and hyphens (-)
+  The valid length is limited from can contain `3` to `255`, only Chinese and English letters, digits,  hyphens (-),  
+  underscores(_),  periods (.),  slashes,  commas in Chinese formats, parentheses and colons in Chinese and English formats
   are allowed.  
-  The name must start with a Chinese or English letter.
+  The name must start with digit, Chinese or English letter.
```
Comment (Lance52259) [inline docs/resources/apig_api.md]:
```suggestion
  The name must start with a digit, Chinese or English letter.
```

File: docs/resources/apig_appcode.md [hunk 2]
```diff
@@ -49,7 +49,8 @@ The following arguments are supported:
   Changing this will create a new resource.
 
 * `value` - (Optional, String, ForceNew) Specifies the APPCODE value (content).
-  The value can contain `64` to `180` characters, starting with a letter, plus sign (+), or slash (/). Only letters and
+  The value can contain `64` to `180` characters, starting with a letter, plus sign (+), or slash (/), or digit.
+  Only letters, digit and
   the following special characters are allowed: `+_!@#$%/=`.
```
Comment (Lance52259) [inline docs/resources/apig_appcode.md]:
```suggestion
  Only letters, digit and the following special characters are allowed: `+_!@#$%/=`.
```

File: docs/resources/apig_instance.md [hunk 3]
```diff
@@ -108,6 +108,9 @@ The following arguments are supported:
 
 * `vpcep_service_name` - (Optional, String) Specifies the name of the VPC endpoint service.
   It can contain a maximum of 16 characters, including letters, digits, underscores (_), and hyphens (-).
+  If this parameter is not specified, the system automatically generates a name in the "{region}.apig.{service_id}" format.
+   If this parameter is specified, the system automatically generates a name in the
+   "{region}.{vpcep_service_name}.{service_id}" format.
```
Comment (Lance52259) [inline docs/resources/apig_instance.md]:
```suggestion
  If this parameter is specified, the system automatically generates a name in the
  "{region}.{vpcep_service_name}.{service_id}" format.
```

File: docs/resources/apig_api.md (hunk 1 above)
Comment (wuzhuanhong) [reply docs/resources/apig_api.md]: fixed

File: docs/resources/apig_appcode.md (hunk 2 above)
Comment (wuzhuanhong) [reply docs/resources/apig_appcode.md]: fixed

File: docs/resources/apig_instance.md (hunk 3 above)
Comment (wuzhuanhong) [reply docs/resources/apig_instance.md]: fixed

**Pull request merged**